hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/resources/mapred-default.xml

```diff
@@ -1660,14 +1660,18 @@
   <name>yarn.app.mapreduce.client-am.ipc.max-retries</name>
   <value>3</value>
   <description>The number of client retries to the AM - before reconnecting
-    to the RM to fetch Application Status.</description>
+    to the RM to fetch Application Status.
+    In other words, it is the ipc.client.connect.max.retries to be used during
+    reconnecting to the RM and fetching Application Status.</description>
```

Comment:
I think the original information is enough, do we need to add a new description?

Reply:
I think it's a good update, because it makes it clear that `ipc.client.connect.max.retries` (the common property for controlling this in Hadoop's RPC framework) is not actually in effect for this code path, and instead there is another property, with a different default, taking the place of `ipc.client.connect.max.retries`.

This also brings this property's documentation into alignment with other similar properties, such as these from yarn-default.xml:

```
  <property>
    <description>When HA is enabled, the number of retries per
      attempt to connect to a ResourceManager. In other words,
      it is the ipc.client.connect.max.retries to be used during
      failover attempts</description>
    <name>yarn.client.failover-retries</name>
    <value>0</value>
  </property>
```

```
  <property>
    <description>When HA is enabled, the number of retries per
      attempt to connect to a ResourceManager on socket timeouts. In other
      words, it is the ipc.client.connect.max.retries.on.timeouts to be used
      during failover attempts</description>
    <name>yarn.client.failover-retries-on-socket-timeouts</name>
    <value>0</value>
  </property>
```

Reply:
@cnauroth Thanks for your explanation, I agree with you.

change LGTM +1.